### PR TITLE
New plasmaman suit - Coke Suit

### PR DESCRIPTION
**Pull request by McRamon**
## What Does This PR Do
Adds new plasmaman suit, available in loadout, not bound to any job and instead brought to you be Space Cola Company.
Suit and helmet come in a box.

## Why It's Good For The Game
Plasmamen lack drip, with this suit first and more suits in future we will be able to expand the variety of clothes for plasmamen.

## Images of changes
image

## Testing
Selected as loadout, joined, put on.

## Changelog
:cl:
add: New clothes option for plasmamen - Coke Suit!
/:cl:

### CODE REVIEW

File: code/modules/client/preference/loadout/loadout_racial.dm [hunk 1]
```diff
@@ -75,3 +75,8 @@
 	description = "Large, comfortable robes worn by those who need a bit more covering. The thick fabric contains a pocket suitable for those that need their hands free during their work, while the cloth serves to cover scars or other injuries to the wearer's body."
 	path = /obj/item/clothing/suit/hooded/vox_robes
 	slot = SLOT_HUD_OUTER_SUIT
+
+/datum/gear/racial/plasmamansuit_coke
+	display_name = "Coke Suit"
+	description = "Plasmaman envirosuit designed by Space Cola Co and gifted to the people of Boron as part of an eleborate advertisement company"
```
Comment (Henri215) [inline code/modules/client/preference/loadout/loadout_racial.dm:81]:
```suggestion
	description = "Plasmaman envirosuit designed by Space Cola Co and gifted to the people of Boron as part of an elaborate advertisement company."
```

Comment (McRamon) [reply code/modules/client/preference/loadout/loadout_racial.dm:81]: damn you fast

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 2]
```diff
@@ -1233,6 +1233,13 @@
 	for(var/I in 1 to 7)
 		new /obj/item/grenade/chem_grenade/metalfoam(src)
 
+/obj/item/storage/box/coke_envirosuit
+	icon_state = "plasma_box"
```
Comment (Henri215) [inline code/game/objects/items/weapons/storage/boxes.dm:1237]: Good idea to give the box a name.

Comment (McRamon) [reply code/game/objects/items/weapons/storage/boxes.dm:1237]: oh true

File: code/modules/client/preference/loadout/loadout_racial.dm [hunk 3]
```diff
@@ -75,3 +75,8 @@
 	description = "Large, comfortable robes worn by those who need a bit more covering. The thick fabric contains a pocket suitable for those that need their hands free during their work, while the cloth serves to cover scars or other injuries to the wearer's body."
 	path = /obj/item/clothing/suit/hooded/vox_robes
 	slot = SLOT_HUD_OUTER_SUIT
+
+/datum/gear/racial/plasmamansuit_coke
+	display_name = "Coke Suit"
+	description = "Plasmaman envirosuit designed by Space Cola Co and gifted to the people of Boron as part of an elaborate advertisement company."
```
Comment (Henri215) [inline code/modules/client/preference/loadout/loadout_racial.dm:81]:
```suggestion
	description = "Plasmaman envirosuit designed by Space Cola Co and gifted to the people of Boron as part of an elaborate advertisement campaign."
```

**Review by Henri215 — COMMENTED** [review]
Everything else is looking good

**Review by S34NW — APPROVED** [review]

**Pull request merged**